requirements.txt

```diff
@@ -3,7 +3,7 @@ bleach==6.0.0
 datacite==1.1.3
 gunicorn==21.2.0
 hepdata-converter-ws-client==0.2.2
-hepdata-validator==0.3.3
+hepdata-validator @ git+https://github.com/HEPData/hepdata-validator@submission-linking
```

Comment:
You can update this to `hepdata-validator==0.3.4` now.  I've checked that, after the upgrade, I can now use [`import-records`](https://hepdata.readthedocs.io/en/latest/cli.html#hepdata-importer-import-records) to import records with large tables given as examples in #136 into my local instance.